pkg/cli/subcommands/check/command.go

```diff
@@ -104,7 +104,6 @@ type cliParams struct {
 	formatJSON                bool
 	formatTable               bool
 	breakPoint                string
-	fullSketches              bool
```

Comment:
Need to remove line 239 as well `cmd.Flags().BoolVar(&cliParams.fullSketches, ...`